source/applications/basic/file_formats/read_iterate_zdf.py

```diff
@@ -21,27 +21,27 @@ def _main():
     frame = zivid.Frame(filename_zdf)
 
     # Extracting point cloud from the frame
-    point_cloud = frame.get_point_cloud().to_array()
-    xyz = np.dstack([point_cloud["x"], point_cloud["y"], point_cloud["z"]])
-    rgb = np.dstack([point_cloud["r"], point_cloud["g"], point_cloud["b"]])
-    contrast = np.dstack([point_cloud["contrast"]])
+    xyzrgba = frame.point_cloud().copy_data("xyzrgba")
+    xyz = np.dstack([xyzrgba["x"], xyzrgba["y"], xyzrgba["z"]])
+    rgb = np.dstack([xyzrgba["r"], xyzrgba["g"], xyzrgba["b"]])
```

Comment:
May I suggest using copy_data("xyz") and copy_data("rgba") separately?

Reply:
done